### PR TITLE
Box filters

### PR DESCRIPTION
**Pull request by cohenlea**
Cleaned up and extracted some variables and a function to make the part where we apply the box filters easier to read. Apparently stacked/multiple ternary operators is not usually a good idea, and pretty impossible to debug.

### CODE REVIEW

**Comment by cohenlea** [comment]
[Fix confusion around "Active Boxes" in Boxes Menu. #small](https://trello.com/c/4Ze0C6WQ/360-fix-confusion-around-active-boxes-in-boxes-menu-small)

File: include/stock.php [hunk 1]
```diff
@@ -22,7 +27,28 @@
         listsetting('manualquery', true);
 
         //dump($listconfig);
```
Comment (HaGuesto) [inline include/stock.php]: remove this line

**Comment by cohenlea** [comment]
ok thanks for the review @HaGuesto !

**Comment by HaGuesto** [comment]
@cohenlea please check the updates in the trello card.

**Comment by cohenlea** [comment]
@HaGuesto ready for review again, thanks

**Comment by cohenlea** [comment]
@HaGuesto I followed your strategy, let me know if this is what we're going for. For consistency, I don't know if you want to restructure yours to have the title/label as "Boxes" and the first filter "In Stock" instead of In Stock at the top label. It's something we discussed and I changed just in this PR.

**Pull request merged**